curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dfa30b9eacea3f48c6300ad.md

```diff
@@ -29,7 +29,7 @@ Your anchor (`a`) element should have an opening tag. Opening tags have this syn
 assert(document.querySelectorAll('a').length >= 2);
 ```
 
-You should only add one opening anchor (`a`) tag. Please remove any extras.
+Creating a link with the anchor element requires both an opening and closing (`a`) tag.
```

Comment:
I don't think this will be a wording only change - we may need to experiment. This text does not actually check if there is both an opening and closing anchor tag, at least not directly, so I think we can generate code that fails this test but this text wouldn't be helpful in resolving the issue

Reply:
Hi @jeremylt, While verifying my changes I checked  Hint message without any anchor tag and results were different from the Hint message with an anchor tag.  So, from the message it seems some handling of number of tags is done. 
Attaching images for reference.

![Screenshot 2023-06-15 at 10 11 57 PM](https://github.com/freeCodeCamp/freeCodeCamp/assets/28575935/334d4c36-6631-4702-9f9c-010365c82ab1)

![Screenshot 2023-06-15 at 10 12 21 PM](https://github.com/freeCodeCamp/freeCodeCamp/assets/28575935/27d32cdc-dc12-453a-88ee-ebe8558e5a8a)

Reply:
Seems good for this too.
![Screenshot 2023-06-15 at 10 23 57 PM](https://github.com/freeCodeCamp/freeCodeCamp/assets/28575935/cbe7d20e-306f-47d1-a2b6-10a0d47a3921)

Reply:
That is *one* anchor element. Add another.

Reply:
my mistake, i picked the task early. pls do drop PR if required